### ISSUE TITLE
Fedora 31 Gnome 3.34.3 Thinkpad X1 Carbon 6th Gen

### ISSUE THREAD

**Issue by alexjfinch** [mpiannucci/gnome-shell-extended-gestures#74]
Hi all,

Seems that the extension doesn't work in Gnome 3.34.3, Fedora 31 on a X1 Carbon 6th gen.

I've seen others using it in Fedora 31 so I'm wondering if its something to do with the laptop.

Cheers
Al

**Comment by mpiannucci** [comment]
Ive seen others use a thinkpad with no issues so that is a new one for sure.

**Comment by rmader** [comment]
On certain touchpads, three and four finger gestures broke after a recent libinput update in Fedora (happened to me, too, on an Thinkpad T460p). For me this was fixed by the update from today, libinput 1.15.3-2. So if the four finger inbuild gestures is also broken for, then you're most likely hitting the same bug and should be fine today or tomorrow.

**Comment by udaemon** [comment]
hi, I can report that on my machine (thinkpad x270 with arch linux and latest gnome deskop, i. e. 3.36) 3 finger horizontal swipes work well whereas vertical swipes with 3 fingers seem to be reversed: swiping up evokes the action I set for a down motion and vice versa. 
another thing I observe is that all actions related to the workspaces don’t to not work, nothing happens on such a setting (while swiping itself works, though reversed as pointed out above).

I have no insights but I can imagine that with the recent update of gnome some definitions concerning the workspaces have changed, that’s why there’s no response to the extended gestures in these cases.

can anybody else confirm this behaviour on their desktop?

**Comment by mpiannucci** [comment]
Yeah the 4 finger gestures are grabbed by Gnome and there is currently no way to override them unfortuantely.

**Comment by rmader** [comment]
> I have no insights but I can imagine that with the recent update of gnome some definitions concerning the workspaces have changed, that’s why there’s no response to the extended gestures in these cases.
> 
> can anybody else confirm this behaviour on their desktop?

Can confirm. But we should open a new issue about it.
